src/mumble/main.cpp

```diff
@@ -364,6 +376,16 @@ int main(int argc, char **argv) {
 				g.bDebugDumpInput = true;
 			} else if (args.at(i) == QLatin1String("--print-echocancel-queue")) {
 				g.bDebugPrintQueue = true;
+			} else if (args.at(i) == "--print-translation-dirs") {
+				printTranslationDirs = true;
+			} else if (args.at(i) == "--translation-dir") {
+				if (i + 1 < args.count()) {
+					extraTranslationDirs.append(args.at(i + 1));
+					i++;
+				} else {
+					qCritical("Missing argument for --translation-dir!");
+					return 1;
```

Comment:
```suggestion
					qFatal("Missing argument for --translation-dir!");
```

Reply:
No- we have the explicit return value at other places as well. It is also important to have that since we might at some point change the side-effects of `qFatal` (shutting down the application) and when this happens the return statements are there to prevent bad things from happening.